canton/community/participant/src/main/daml/Canton/Internal/Bong.daml

```diff
@@ -6,8 +6,6 @@ module Canton.Internal.Bong where
 -- import Daml.Script
 import DA.Text
 import DA.List
-import DA.Time
-import DA.Date
```

Comment:
Thanks @paulbrauner-da , so these we will need to remove or comment out in the canton-repo, right?